### PR TITLE
feat(front): invert all icons

### PR DESCRIPTION
**Pull request by knodelle**
Issue: #5511 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Comment by knodelle** [comment]
Instead of using 

`[dir=rtl] .fa`

I used : 

`[dir=rtl] i`

Because the CSS does not detect icons with only .fa

Is it ok to use it like that?

File: frontend/src/css/IconsRtl.styl [hunk 1]
```diff
@@ -1,14 +1,3 @@
...
-[dir=rtl] .fa-share-alt
-[dir=rtl] .fa-sign-out-alt
-[dir=rtl] .fa-cogs
-[dir=rtl] .fa-tag
-[dir=rtl] .fa-magic
-[dir=rtl] .fa-folder-open
-[dir=rtl] .fa-folder
-[dir=rtl] .fa-comments
-	transform scale(-1,1)
+[dir=rtl] i
```
Comment (gbonaspetti) [inline frontend/src/css/IconsRtl.styl:2]: You need to update the issue if you are not gonna use .fa

Comment (knodelle) [reply frontend/src/css/IconsRtl.styl:2]: done, thanks :)

**Review by gbonaspetti — APPROVED** [review]

**Comment by grignards** [comment]
> Instead of using
> 
> `[dir=rtl] .fa`
> 
> I used :
> 
> `[dir=rtl] i`
> 
> Because the CSS does not detect icons with only .fa
> 
> Is it ok to use it like that?

Err… *<i>* is normally used to emphasize a content (in general it will render in italic). I'm afraid that with this change the italic texts in tinymce (for example) will be reversed…

File: frontend/src/css/IconsRtl.styl (hunk 1 above)
Comment (grignards) [inline frontend/src/css/IconsRtl.styl:2]:
```suggestion
[dir=rtl] .fas,.far,.fal,.fad,.fab
```

**Review by grignards — CHANGES_REQUESTED** [review]

**Comment by grignards** [comment]
Tested on branch.

File: frontend/src/css/IconsRtl.styl (hunk 1 above)
Comment (grignards) [reply frontend/src/css/IconsRtl.styl:2]: This selector has the same effect but is more specific to font awesome icons styling.

**Comment by knodelle** [comment]
the test login/login_after_disconnection_spec.js failed, it is unstable. I will reload and if it still fails I will add it to an issue to fix it

**Pull request merged**